manual/mapper/entities/README.md

```diff
@@ -159,10 +159,43 @@ private int day;
 This information is used by some of the DAO method annotations; for example,
 [@Select](../daos/select/)'s default behavior is to generate a selection by primary key.
 
+#### Computed properties
+
+Annotating an entity property with [@Computed] indicates that when retrieving data with a 
+[@Select]-annotated method that this property should be set to the result of a computation on the
+Cassandra side, typically a function call:
+
+```java
+private int v;
+
+@Computed("writetime(v)")
+private int writeTime;
+```
+
+The CQL return type of the formula must match the type of the property, otherwise an exception 
+will be thrown.
+
+[@Computed] does not support case-sensitivity. If the expression contains case-sensitive column 
+or function names, you'll have to escape them:
+
+```java
+@Computed("\"myFunction\"(\"myColumn\")")
+private int f;
+```
+
+Finally, note that [@Computed] fields are only used for [@Select]-annotated dao methods. Both 
```

Comment:
Is this ok? Mapper 3.x was similar with `@Computed` not working with `@Accessor`, for which we opened [JAVA-832](https://datastax-oss.atlassian.net/browse/JAVA-832).   I'm not sure there is an easy way to resolve this without changing query contents.

One possible way to make that work is to alias the result to the name of the property.  We can then instruct others that when they make their queries to do the same, i.e.:

```
@Computed("ttl(x)")
private int xttl;
```

Would cause our select query to query this as `ttl(x) as xttl` instead of what i'm currently doing which is `ttl(x) as "[alias0]"`.

Reply:
Yeah, I think it would make sense to use `getCqlName()` as the alias, and require that people do the  same in their custom queries.

Reply:
Awesome, definitely like this solution, its much more pleasant and also solves my other problem in `EntityHelperGetMethodGenerator` and also opens up computed to be used with the other annotations :+1:

Reply:
Took a stab at that in 3767374 and I think it's a much more elegant solution overall.

I also made it work such that naming strategy / and @CqlName are regarded.

Updated tests to verify `@Query` and `@GetEntity` work assuming you make the right query, and updated the documentation to give instruction on how to do that.